python-sdk/src/astro/databases/aws/redshift.py

```diff
@@ -129,6 +129,19 @@ def table_exists(self, table: BaseTable) -> bool:
         inspector = sqlalchemy.inspect(self.sqlalchemy_engine)
         return bool(inspector.dialect.has_table(self.connection, table.name, schema=table.metadata.schema))
 
+    @staticmethod
+    def get_table_qualified_name(table: BaseTable) -> str:  # skipcq: PYL-R0201
```

Comment:
@pankajastro Why did we implement this method here? Should come from the base class?

Reply:
I want to have a different implementation for redshift.  Redshift accepts the table name like ```"db"."schema"."table"``` this is more safe. 
If I want a user to allow to run the query ```select * from db.schema.table``` in the current scenario if a user will pass the table name "table LIMIT 5" then the query will become ```select * from db.schema.table LIMIT 5``` which is incorrect but if we wrap the table name with a double quote then it would be more correct like ```select * from "db"."schema"."table LIMIT 5"```

Reply:
We should address this issue separately
ref: https://docs.aws.amazon.com/redshift/latest/dg/r_enable_case_sensitive_identifier.html

Reply:
issue: https://github.com/astronomer/astro-sdk/issues/1212